src/SourceGen/ExpressionComparerGenerator.cs

```diff
@@ -26,8 +28,9 @@ public void Execute(GeneratorExecutionContext context)
             var code = ExpressionVisitorGenerator.GenerateVisitor(spec);
             context.AddSource($"{spec.TypeName}.g.cs", code);
 
-            System.IO.Directory.CreateDirectory(@"C:\dev\sourcegenout");
-            System.IO.File.WriteAllText($@"C:\dev\sourcegenout\{spec.TypeName}.cs", $"// Generated {DateTime.Now}{Environment.NewLine}" + code);
+            var sourcegenoutDirectory = Path.Combine(RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? @"C:\" : "/tmp", "dev", "sourcegenout");
```

Comment:
@madelson Build was failing on AppVeyor, which I think is because `/dev` is a special folder on Linux. For now I resorted to `/tmp`, but that made me wonder whether we should use `Path.GetTempPath()`; thoughts?